packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -83,13 +85,19 @@ export interface ISiteState {
     formView: boolean;
     devToolsView: boolean;
     locale: string;
+    theme: IThemeValues;
 }
 
 export enum SiteSlot {
     category = "category",
     categoryIcon = "category-icon"
 }
 
+export interface IThemeValues {
```

Comment:
I don't think "Values" adds ... ironically... value here, I think it can just be `ITheme`

Reply:
+1